.drone.yml

```diff
@@ -44,17 +44,17 @@ trigger:
 
 steps:
 - name: build
-  image: grafana/grafana-plugin-ci:1.3.0-alpine
+  image: grafana/grafana-plugin-ci:1.6.1-alpine
   commands:
   - mage -v build
 
 - name: lint
-  image: grafana/grafana-plugin-ci:1.3.0-alpine
+  image: grafana/grafana-plugin-ci:1.6.1-alpine
   commands:
   - mage -v lint
 
 - name: test
-  image: grafana/grafana-plugin-ci:1.3.0-alpine
+  image: grafana/grafana-plugin-ci:1.6.1-alpine
   commands:
   - mage -v testRace
```

Comment:
@toddtreece -- any luck updating the signature?

I am not having any luck 😬 , but drone will let me build by manually clicking OK

Reply:
pushed with [9648b2b](https://github.com/grafana/grafana-plugin-sdk-go/pull/585/commits/9648b2bb09410cd60b608b0b3a87e9c0e898b81c)

Reply:
thank you! thank you!